src/ocl/gcn_asm_utils.cpp

```diff
@@ -80,6 +92,10 @@ std::string GetGcnAssemblerPathImpl()
         return CleanupPath(asm_path_env_p.c_str());
     }
 #ifdef MIOPEN_AMDGCN_ASSEMBLER // string literal generated by CMake
+#ifdef MIOPEN_CMP_FLAGS_NEW
+    /* Initialize MIOPEN Compiler Flags */
+    MIOPEN_CMP_FLAGS_INIT();
```

Comment:
If this is the only place the code is used, then put the code here rather than defining it in a header file and then expanding it here.

Reply:
Updated version has updates with avoiding INIT style operations